Source/iOS/TwoWayBinding.swift

```diff
@@ -38,7 +38,7 @@ private func nonMarkedText(_ textInput: UITextInput) -> String? {
     return (textInput.text(in: startRange) ?? "") + (textInput.text(in: endRange) ?? "")
 }
 
-public func <-> <Base>(textInput: TextInput<Base>, variable: Variable<String>) -> Disposable {
+public func <-> <Base>(textInput: TextInput<Base>, variable: BehaviorRelay<String>) -> Disposable {
```

Comment:
```suggestion
public func <-> <Base>(textInput: TextInput<Base>, relay: BehaviorRelay<String>) -> Disposable {
```

Reply:
@freak4pc renamed all instances! please review it again.